public-ui/src/environments/test-env-base.ts

```diff
@@ -6,5 +6,6 @@ export const testEnvironmentBase = {
   tcellapikey: 'AQEBBAEkx4iE2KxNyI7Wx08EwU1ycTM7E4FMSmaibbMUQxNU6uQvuAJt7fyABAtFYSYfgEE',
   publicApiUrl: 'https://api-dot-aou-db-test.appspot.com',
   workbenchUrl: 'https://all-of-us-workbench-test.appspot.com',
+  researchAllOfUsUrl: 'https://staging.researchallofus.org',
```

Comment:
nit: can now remove this from `environment.ts` and `environment.local.ts` since they extend this base environment.

Reply:
thanks for the help